modules/planning/learning_based/model_inference/model_inference_test.cc

```diff
@@ -14,16 +14,19 @@
  * limitations under the License.
  *****************************************************************************/
 
-#include "cyber/common/file.h"
-#include "gflags/gflags.h"
 #include "gtest/gtest.h"
+
```

Comment:
Why blank line here?

Reply:
I'm not sure. vscode format this automatically :)

Reply:
I got it now, "gtest/gtest.h" is the header file and the other one is third-party file, so there should be a blank line in between